### DISCUSSION TITLE
Sony Bravia working great

### DISCUSSION THREAD

**Discussion by matticas** [giejay/Immich-Android-TV#3] [General]
Hi there;
Thanks a million for this app!  We previously used Emby which was rather slow to navigate on our setup.
Using a Sony Bravia thing, about 2020/2021 model, app installed fine, logged in first go, everything loaded.  Can navigate all albums, open and scroll photos. 
Two notes; 

1. the title at the top, and slightly darker transparent bar at the top when fulling a full sized photo doesn't go away.
2. slideshow / play doesn't work.  Had it set to 3 seconds I think.  I can trigger the play/pause action/animation of the icon, but the photos don't scroll.

Thanks again!

**Comment by giejay** [comment]
1. Thats a setting to show the title and the item count. You can change it in view settings of screensaver settings.
2. Thats strange, on what version are you? You can check it in settings-server. I did fix some stuff in the latest version (1.16), but not playing it all is weird. How many items do you see being displayed in the top transparant bar. Can you reproduce it on the demo environment?

**Reply by matticas** [reply]
On version 1.16, just downloaded an hour ago or so.
Let me re-check

**Reply by matticas** [reply]
Ok;
I haven't tested the screensaver yet, but I have enabled it.
Under; Settings; View Settings I selected 3 seconds.
Went into an album, and the title bar dark transparent thing has filename and number / total-album-number.  
Pushing enter on the remote toggles play/pause animation but what I would call reverse.  It actually does scroll (I thought it was paused earlier).  
When it's playing; I push enter, the Play symbol flashes up - I would think it should be pause.  
When I push enter again from the paused state, the pause symbol flashes up, and resuming photo scrolling.
